### PR TITLE
Code improvements

### LINKED ISSUE #17: Extract metamask interaction methods from index.ts
Maybe have `src/metamask` directory with every method in separate file?

### LINKED ISSUE #18: getMetamask
now `getMetamask` has `confirmWelcomeScreen` `importAccount` `closeNotificationPage` (propably shoud be `prepareMetamask`) what arent actually part of getting metamask and getting metamask methods

### PR DESCRIPTION
**Pull request by BeroBurny**
closes #17 
closes #18

### CODE REVIEW

File: src/metamask/index.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import { lock } from "./lock";
+import { sign } from "./sign";
+import { switchAccount } from "./switchAccount";
+import { switchNetwork } from "./switchNetwork";
+import { unlock } from "./unlock";
+
+export type SetSignedIn = (state: boolean) => Promise<void>;
+export type GetSingedIn = () => Promise<boolean>;
+
+export const getMetamaskMethods = async (page: Page, version?: string): Promise<Dappeteer> => {
```
Comment (mpetrunic) [inline src/metamask/index.ts:18]:
```suggestion
export const getMetamask= async (page: Page, version?: string): Promise<Dappeteer> => {
```

**Review by mpetrunic — COMMENTED** [review]
a lot of lint errors

File: src/metamask/index.ts (hunk 1 above)
Comment (BeroBurny) [reply src/metamask/index.ts:18]: `getMetamask` is in the root that why I named it `getMetamaskMethods` and this function only return methods when the root find MetaMask window and call this function to get methods for window

File: .eslintrc.js [hunk 2]
```diff
@@ -1,77 +1,78 @@
...
+  parser: '@typescript-eslint/parser',
+  plugins: ['@typescript-eslint', 'eslint-plugin-import', 'prettier'],
+  extends: [
+    'eslint:recommended',
+    'plugin:@typescript-eslint/recommended',
+    'prettier/@typescript-eslint',
+    'plugin:prettier/recommended',
+  ],
+  rules: {
+    '@typescript-eslint/no-require-imports': 'error',
```
Comment (mpetrunic) [inline .eslintrc.js:17]: add "prettier/prettier": ["error", {}, { usePrettierrc: true }], rule

File: src/index.ts [hunk 3]
```diff
@@ -1,285 +1,144 @@
...
-    ],
-    ...rest
-  })
+    args: [`--disable-extensions-except=${METAMASK_PATH}`, `--load-extension=${METAMASK_PATH}`, ...(args || [])],
+    ...rest,
+  });
 }
 
-export async function getMetamask(
+export async function setupMetamask(
```
Comment (mpetrunic) [inline src/index.ts:51]: could you add comment on that method that it will setup metamask account. Maybe also change return type to be Metamask methods?

**Review by mpetrunic — APPROVED** [review]

**Review by mpetrunic — APPROVED** [review]

**Pull request merged**